inference-engine/tests/functional/plugin/shared/include/behavior/plugin/core_integration.hpp

```diff
@@ -886,6 +886,12 @@ TEST_P(IEClassQueryNetworkTest, QueryNetworkHETEROWithBigDeviceIDThrows) {
 // LoadNetwork
 //
 
+TEST_P(IEClassNetworkTestP, LoadNetworkWithoutDeviceNameNoThrow) {
+  SKIP_IF_CURRENT_TEST_IS_DISABLED()
+  InferenceEngine::Core  ie = BehaviorTestsUtils::createIECoreWithTemplate();
+  ASSERT_NO_THROW(ie.LoadNetwork(actualCnnNetwork));
+}
```

Comment:
@myshevts should not we add tests with performance hints? It's more important to test with hints with compare to empty config